REQS.md

```
@@ -684,13 +685,16 @@ This section documents features that are standard in competing pickleball/sports
 | First-Come-First-Served | First to accept gets the slot |
 | Expiring Offers | Time limit to accept (e.g., 15 minutes) |
 | Auto-Decline | Remove from waitlist if not accepted in time |
+<!-- END WIP -->
 
+<!-- BEGIN WIP: STORY-0034 -->
 **Configuration Options:**
 | Setting | Description |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)